lib/committee/schema_validator/hyper_schema.rb

```diff
@@ -69,6 +69,7 @@ def request_unpack(request)
             allow_query_params: validator_option.allow_query_params,
             coerce_form_params: validator_option.coerce_form_params,
             optimistic_json:    validator_option.optimistic_json,
+            allow_get_body:     validator_option.allow_get_body,
```

Comment:
Can you alphabetize this by moving `allow_get_body` up above `allow_query_params`?